MANIFEST.in

```
@@ -3,23 +3,8 @@ include README.md
 
 include setupbase.py
 
-include package.json
-include *.config.js
-include setupJest.js
-include ts*.json
 include jupyterlab_git/labextension/*.tgz
```

In #461 I have seen that we are missing the server extension configuration file.

```suggestion
include jupyterlab_git/labextension/*.tgz
recursive-include jupyter-config *.json
```

---

Good catch

---

After looking into it, I discovered that neither of these lines are needed in `MANIFEST.in`:

```
include jupyterlab_git/labextension/*.tgz
recursive-include jupyter-config *.json
```

since they're both already handled/included internally in `setup.py` by the `data_files` mechanism. You can prove this to yourself by looking at the verbose output of various `pip`/`python setup.py` commands, or by just looking in the concrete `MANIFEST` file that gets generated when you run `python setup.py sdist` command.

---

Thanks for the analysis and sharing the knowledge.